spark/src/main/scala/org/apache/mahout/sparkbindings/io/MahoutKryoRegistrator.scala

```diff
@@ -35,7 +35,5 @@ class MahoutKryoRegistrator extends KryoRegistrator {
     kryo.addDefaultSerializer(classOf[Vector], new WritableKryoSerializer[Vector, VectorWritable])
     kryo.addDefaultSerializer(classOf[DenseVector], new WritableKryoSerializer[Vector, VectorWritable])
     kryo.addDefaultSerializer(classOf[Matrix], new WritableKryoSerializer[Matrix, MatrixWritable])
-    kryo.register(classOf[com.google.common.collect.HashBiMap[String, Int]], new JavaSerializer())
-
   }
```

Comment:
Took out BiDictionary from Kryo per @dlyubimov Not completely sure why this is bad except *any* mods to Spark/Kryo setup can bite you later. This means that serialization of the dictionaries is not as fast as it could be but make Mahout easier to integrate with applications as a library, though not much because you still have to make sure this method get called before a context is created.